src/cameras/OrthographicCamera.js

```diff
@@ -38,22 +38,19 @@ THREE.OrthographicCamera.prototype.updateProjectionMatrix = function () {
 
 THREE.OrthographicCamera.prototype.clone = function () {
 
-	var camera = new THREE.OrthographicCamera();
+	var camera = new this.constructor( this.left, this.right, this.top, this.bottom, this.near, this.far );
+	return camera.copy( this );
 
-	camera.copy( this );
-
-	camera.zoom = this.zoom;
-
-	camera.left = this.left;
-	camera.right = this.right;
-	camera.top = this.top;
-	camera.bottom = this.bottom;
-
-	camera.near = this.near;
-	camera.far = this.far;
-
-	return camera;
+};
 
+THREE.OrthographicCamera.prototype.copy = function ( source ) {
+	
+	THREE.Camera.prototype.copy.call( this, source );
+	
+	this.zoom = source.zoom;
```

Comment:
It's probably best to copy `.left`, `.right`, etc in `copy()`. They are set in `clone()`, but what if someone doesn't doesn't call `clone()` first, e.g. using `copy()` to make to camera equal?